tests/test_mappings.py

```diff
@@ -10,6 +10,7 @@
 
 from . import get_data_files, get_fp_data_files
 from detection_rules.utils import combine_sources, evaluate, load_etc_dump
+from detection_rules.rule_loader import RULES_CONFIG
```

Comment:
Might be good to double check that this is the best import to obtain the `RULES_CONFIG` so we don't create unnecessary circular dependencies in the future (and if there is any consistency that makes sense based on how we import in the other tests files).

Reply:
Agreed, initially I pulled it from the Rule Loader to follow convention with where we pulled it in devtools.py `from .rule_loader import RULES_CONFIG`. 

The more widely used approach is to always parse the config again on initialization (since it is cached there is just a cache lookup rather than duplicate parsing) 
```
from .config import parse_rules_config

RULES_CONFIG = parse_rules_config()
```

Perhaps we should standardize to one or the other to avoid confusion? IMO the more direct we go with importing the better, to reduce the chance circular importing. Meaning, I think we should use the later option even though it is an additional line of code.